scripts/install-multi-user.sh

```diff
@@ -56,6 +56,9 @@ readonly NIX_INSTALLED_CACERT="@cacert@"
 #readonly NIX_INSTALLED_CACERT="/nix/store/7dxhzymvy330i28ii676fl1pqwcahv2f-nss-cacert-3.49.2"
 readonly EXTRACTED_NIX_PATH="$(dirname "$0")"
 
+# allow to override identity change command
+readonly NIX_BECOME=${NIX_BECOME:-sudo}
```

Comment:
I wonder if we could document this somewhere?

Reply:
Before documenting this as officially supported, need to ensure that alternatives work as expeted. Which requires CI test setup with Arch + `doas`, for example.